addons/website_forum/controllers/main.py

```diff
@@ -164,7 +164,7 @@ def forum_faq(self, forum, **post):
     @http.route('/forum/get_tags', type='http', auth="public", methods=['GET'], website=True, sitemap=False)
     def tag_read(self, query='', limit=25, **post):
         data = request.env['forum.tag'].search_read(
-            domain=[('name', '=ilike', (query or '') + "%")],
+            domain=[('name', '=ilike', (query or '') + "%"), ('forum_id', '=', int(post.get('forum_id', False)))],
```

Comment:
@rdeodoo Thank you for the suggestion, I've made the necessary changes. Please go through it again. Because 'forum id' returns type str, it must be transformed to int in order to search. Thank you !

Reply:
You forgot the `TODO` comment

And also
> Also if you don't mind wrapping your last commit line as you did with the previous lines :)

```
Solution: When adding tags to posts, identify whether the new tag belongs to the current forum or not. If not on the current forum, then create new. If it belongs, then update the tag's id to the post
```

```
Solution: When adding tags to posts, identify whether the new tag
belongs to the current forum or not. If not on the current forum, then
create new. If it belongs, then update the tag's id to the post
```

This is for the commit message to be easily readable, otherwise you have to scroll horizontally on `tig` or `git` which is **really** annoying, best to avoid it :)